urdf/uuvsim_teledyne_whn_robot.xacro

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
```

Comment:
Some of these .xacro files have their execute bit set.

Reply:
Fixed in latest commit.

Reply:
Done.